src/docs-ui/app/app.component.scss

```diff
@@ -0,0 +1,21 @@
+// Icons - Swap based on app theme
```

Comment:
Is there a better place to put this? Tried for a while to make it work in _overrides but no joy.

Reply:
Yeah, we need to put it in one of those files to be able to distribute it. 

_overrides didnt work?

Reply:
Nope, has to do with the cascade of the component SCSS I think - :host works in any component theme file in the chain, but not more general global css like _overrides or theme.scss.

Looked into view encapsulation for this but that also seems to only apply to components.

Reply:
Nevermind, got it